src/scss/layouts/_logoSection.scss

```
@@ -1,4 +1,7 @@
 .logoSection {
   width: 30%;
   border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

```suggestion
.logoSection {
 display: flex;
  justify-content: center;
  align-items: center;
  width: 30%;
  border: 1px solid black;
```
Osobiście tak bym to ułożył, reszta super 🗡️

---

Popieram

---

Szanuję za ogarnianie tej kolejności :D